CHANGELOG-3.0.md

```diff
@@ -55,7 +55,7 @@ To get the diff between two versions, go to https://github.com/symfony/symfony/c
  * bug #19127 [Form] Add exception to FormRenderer about non-unique block names (enumag)
  * bug #19118 [Process] Fix pipes cleaning on Windows (nicolas-grekas)
  * bug #19128 Avoid phpunit 5.4 warnings on getMock (2.7+) (iltar)
- * bug #19114 [HttpKernel] Dont close the reponse stream in debug (nicolas-grekas)
+ * bug #19114 [HttpKernel] Dont close the response stream in debug (nicolas-grekas)
```

Comment:
These file are auto-generated on release, the typo comes from the PR actually ;-) see https://github.com/symfony/symfony/pull/19114